docs/machine-learning/how-to-guides/train-machine-learning-model-ml-net.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "How-To: Train and evaluate a machine learning model in ML.NET" 
+description: Learn how to train and evaluate machine learning models in ML.NET
+ms.date: 04/17/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc,how-to
+#Customer intent: As a developer I want to know how to train and evaluate a machine learning model with ML.NET
+---
+
+# How-To: Learn how to train and evaluate a machine learning model in ML.NET
+
+Learn how to build machine learning models, extract learned parameters and measure performance with ML.NET. Although this sample trains a regression model, the concepts are applicable throughout a majority of the other algorithms.
+
+## Split data for training and testing sets
+
+The goal of a good machine learning model is to generalize well or predict better on unseen observations than the initial observations it was trained on. To do so, split data into training and test sets. 
```

Comment:
>better [](start = 75, length = 6)

I'd remove better, because it seems like there are already predictions, and we're trying to make them better.